.github/workflows/ci-e2e.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - 378_add-tls-policy
```

Comment:
Since these changes can't be tested on this PR, it's running here https://github.com/Kuadrant/multicluster-gateway-controller/actions/runs/5832446191/job/15817886112